src/main/java/org/scijava/log/DefaultLogFormatter.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Please remove the conflicts list from the commit message. It's not relevant to the actual patch, only some transient rebase.